src/gitpod.ts

```diff
@@ -41,6 +43,54 @@ export interface GitpodDocker {
   readonly file?: string;
 }
 
+/**
+ * Configure the Gitpod App for prebuilds
+ * Currently the App only support GitHub.
+ * See: https://www.gitpod.io/docs/prebuilds/
+ */
+export interface GitpodPrebuildsConfig {
```

Comment:
No need to use the word "Config" - everything is a config.

```suggestion
export interface GitpodPrebuilds {
```